mappings/net/minecraft/client/gui/widget/TextFieldWidget.mapping

```diff
@@ -31,7 +31,7 @@ CLASS net/minecraft/class_342 net/minecraft/client/gui/widget/TextFieldWidget
 	METHOD charTyped (CI)Z
 		ARG 1 chr
 		ARG 2 keyCode
-	METHOD isMouseOver (DD)Z
+	METHOD mouseOver (DD)Z
```

Comment:
The getters should stay with `is` iirc (may be wrong, did standard change?)

Reply:
No. Getters with an `is` are fine.

Reply:
Is this method supposed to be an override of `Element.isMouseOver`?

Reply:
Fixed